charts/argo-templates/scripts/diff-bq-table.sh

```diff
@@ -15,6 +15,12 @@ for c in ${COMPARE_COLS//,/ }; do
 done
 declare -r FULL_DIFF=$(join_by ' OR ' "${COMPARISONS[@]}")
 
+declare -a PRIMARYCOLUMNS=()
```

Comment:
A little more readable:
```suggestion
declare -a PRIMARY_COLUMNS=()
```